### PR TITLE
net-analyzer/tcping: fix HOMEPAGE and SRC_URI, EAPI=7

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This Bug fixes the HOMEPAGE and SRC_URI from net-analyzer/tcping. I've also updated to EAPI=7 but since this ebuild doesn't have any stable keyword didn't made a revision bump.
Please review.

Closes: https://bugs.gentoo.org/679922
Signed-off-by: Michael Mair-Keimberger <m.mairkeimberger@gmail.com>

### CODE REVIEW

File: net-analyzer/tcping/tcping-1.3.5.ebuild [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-EAPI=4
+EAPI=7
 
 inherit toolchain-funcs
 
 DESCRIPTION="Ping implementation that uses the TCP protocol"
-HOMEPAGE="http://www.linuxco.de/tcping/tcping.html"
-SRC_URI="http://www.linuxco.de/tcping/${P}.tar.gz"
+HOMEPAGE="https://github.com/mkirchner/tcping"
+SRC_URI="https://github.com/mkirchner/tcping/archive/master.tar.gz -> ${P}.tar.gz"
```
Comment (a17r) [inline net-analyzer/tcping/tcping-1.3.5.ebuild]: This change is clearly invalid; it disguises a master snapshot under an existing version number, and if tomorrow there is a bump in this repo the tarball will change again.

**Comment by mm1ke** [comment]
@a17r 
Thanks for the explanation. I've updated the ebuild to choose a specific COMMIT because unfortunately there is no tag or release for this package. Hope i got it correct.

**Comment by a17r** [comment]
That's not going to cut it, it is still not clear how this differs from release 1.3.5 tarball. And the github repository just looks like a one-time code dump anyway, no history, with no activity in a long time. Does upstream react on issues (such as requesting release tarball for tcping)? Our mirrors contain 1.3.5 anyway, so there is no urgency to change that.

**Comment by mm1ke** [comment]
Well, this was fixed in the meantime by jer so no need to look further into this, i'll close this. Though, thanks for the review.

**Pull request closed without merging**